src/lib/bootloader/proposal_client.rb

```diff
@@ -7,6 +7,38 @@
 module Bootloader
   # Proposal client for bootloader configuration
   class ProposalClient < ::Installation::ProposalClient
+    # Error when during update media is booted by different technology than target system.
+    class MismatchBootloader < RuntimeError
+      include Yast::I18n
+
+      def initialize(old_bootloader, new_bootloader)
+        @old_bootloader = old_bootloader
+        @new_bootloader = new_bootloader
+
+        super("Mismatching bootloaders")
```

Comment:
Do we display the exception somewhere? Is it OK to use an untranslated string?

Reply:
no, it is catched in only place where it is available. And if we display it, then it uses user_message.